charts/ckan/templates/ckan/deployment.yaml

```diff
@@ -0,0 +1,166 @@
+{{- $name := "ckan" -}}
+{{- $claimName := printf "%s-%s" (include "common.names.fullname" $) $name -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ printf "%s-%s" (include "common.names.fullname" $) $name | quote }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: {{ $name }}
+spec:
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+  selector:
+    matchLabels: {{- include "common.labels.matchLabels" . | nindent 6 }}
+      app.kubernetes.io/component: {{ $name }}
+  template:
+    metadata:
```

Comment:
Missing annotation with a secret checksum for auto-rotation on secret changes, see https://github.com/teutonet/teutonet-helm-charts/blob/main/charts/base-cluster/templates/monitoring/kube-prometheus-stack/_grafana-config.yaml#L21

Reply:
i have added it there. however, my problem is that i am referencing 3 secrets there. this is not possible with the common.utils.checksumTemplate function

Reply:
That's true, but you can just add multiple annotations 😉

Reply:
is now joined as a checksum using `-`